docs/_source/tutorials_and_integrations/tutorials/feedback/end2end_examples/add-suggestions-and-responses-005.ipynb

```diff
@@ -489,9 +489,7 @@
    "source": [
     "## Conclusion\n",
     "\n",
-    "In this tutorial, we have seen how we can add responses and suggestions to our dataset. Adding responses to your dataset given that you already have the annotated labels is a great way to save time and effort for your project. Similarly, adding suggestions to your dataset will make the annotation process much faster and easier for your annotators. We have seen how we can employ these tools for both `FeedbackDataset` and `RemoteFeedbackDataset`.\n",
-    "\n",
-    "***ADD LINK FOR NEXT TUTORIAL***"
+    "In this tutorial, we have seen how we can add responses and suggestions to our dataset. Adding responses to your dataset given that you already have the annotated labels is a great way to save time and effort for your project. Similarly, adding suggestions to your dataset will make the annotation process much faster and easier for your annotators. We have seen how we can employ these tools for both `FeedbackDataset` and `RemoteFeedbackDataset`. Now, with the dataset we obtained, we can move on to [training our model](/train_model) and [computing the metrics](/compute_metrics). For more detailed info on how to utilize various tools, please refer to our [practical guides](../../../../practical_guides/practical_guides.md)"
```

Comment:
Hi @kursat, these referenced to "/train_model" and "/compute_metrics" are not there yet. Could you create empty notebooks with the correct naming format instead?